KRNumber3/KRNumber3/main.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <string.h>
+#include <stdbool.h>
+
+bool scanFile(char fileName[], char arrayForTest[]) {
```

Comment:
```suggestion
bool scanFile(const char fileName[], const char arrayForTest[]) {
```
Мы их вроде ниже не меняем